src/entity/Alerts.ts

```diff
@@ -11,9 +13,13 @@ export class Alerts {
 
   @Column({ type: 'timestamp without time zone', default: null })
   rankLastSeen: Date | null;
+
+  @Column({ type: 'enum', enum: ['default', 'manual'], default: 'default' })
```

Comment:
According to my experience enums can be very strict and changing them requires some hassle.
My suggestion is to go with a string or number.
We can define a TypeScript enum that translates into a number (like every enum) and we will keep the number in the database. Whatever you want I just want it to be dynamic enough.

Reply:
Ok that makes total sense, will work on that change